components/metal/BuildingSection.vue

```diff
@@ -4,38 +4,43 @@
       <h2 class="building-section__title">
         Building together
       </h2>
-      <p class="building-section__description copy__paragraph_importance_outstanding">
+      <p class="building-section__text">
         Whether you are learning, teaching or performing scientific research, we hope Metal will get you there faster.
       </p>
-
       <div class="building-section__content">
         <div class="building-section__copy">
           <h3 class="copy__subtitle">
             Call for community participation
           </h3>
-          <p class="copy__paragraph">
+          <p class="building-section__text">
             This may just be the beginning, but we think Qiskit Metal has immense potential. We hope to form this software into one usable by superconducting quantum hardware designers across industry and academia, and perhaps even for other qubit architectures. Over time, we hope Metal will become the gateway for conventional microchip designers to get their feet wet designing quantum processors. And we hope that soon, quantum hardware experts will be able to write their own quantum analysis plugins and run them as their own Qiskit Metal backends.
           </p>
           <h3 class="copy__subtitle">
             Open source
           </h3>
-          <p class="copy__paragraph">
+          <p class="building-section__text">
             Qiskit Metal is a visionary work in progress in its early development stages. We‘re working toward a vision—a quantum design & analysis platform built from the ground up specifically for quantum hardware. It is crucial that we keep this product open source in order to grow Qiskit Metal into the software that will be most useful for quantum hardware engineers around the world and for the innovation in and growth of the field as a whole. If Qiskit Metal doesn't have what you want, then you extend Metal to build it!
           </p>
           <h3 class="copy__subtitle">
             Education
           </h3>
-          <!-- eslint-disable vue/multiline-html-element-content-newline -->
-          <p class="copy__paragraph">
-            A few sentences here about educational material. The first link is to “<AppLink
-              class="building-section__copy-link copy__link "
+          <p class="building-section__text">
```

Comment:
The one reason why I had the `eslint-disable` was to prevent this spacing issue, happening in the `AppLink` component: 

Is there a better way of handling this?

![Uploading Screen Shot 2020-10-13 at 7.30.10 PM.png…]()

Reply:
Right! That's why I asked and did not require the change. The correct way to do it is [to add `AppLink` to the ignore list of the rule](https://eslint.vuejs.org/rules/multiline-html-element-content-newline.html#options) because it should be considered an inline element. You can modify the rule or simply restore the previous version, perhaps adding a clarifying comment a-la:
```html
<!-- Disabling linter to keep the quotations and phrasing content together. -->
```

Reply:
Could you add the screenshot again? I can't see it and I'm not sure about the spacing issue you are talking about

Reply:
I've reverted my changes and added the comment you suggest